### DISCUSSION TITLE
Edit Files

### DISCUSSION THREAD

**Discussion by bradmanwalden** [community/community#6835] [Mobile] [answered]
I think a simple text editor, where the save button commits and pushes (similar to the Github Website) would be extremely useful for edits on the go.

**Comment by delameter** [comment]
Agreed.
Preview option when editing markdown files would be also very useful.

**Reply by 13364174888** [reply]
j

**Comment by brianlovin** [comment]
Agreed! I've added your +1 to our tracking issue for these things — both the markdown preview, and ability to make changes to a file + commit directly from mobile.

**Comment by fishooo0815** [comment]
I want to Edith Text Files with GitHub mobile.

**Reply by mestir** [reply]
- [ ] @

**Reply by mestir** [reply]
- [ ]

**Comment by benhatsor** [comment]
A related project is [codeit.codes](https://codeit.codes), from which you can edit and run GitHub files on mobile.

**Comment by spaltrowitz** [comment, marked as answer]
thanks for your suggestion @bradmanwalden! this has now shipped https://github.blog/changelog/2022-11-15-github-for-ios-edit-files-from-browse-code

**Reply by Wladefant** [reply]
Hey, will it be shipped to Android?

**Reply by unique-EJ** [reply]
> Hey, will it be shipped to Android?

**Android** - add "Open with" (Acode, ...) to do this.

![Screenshot_20230214_230057_Android System.png](https://user-images.githubusercontent.com/91560320/218884161-138b54bd-9657-45ac-a589-d75c1b185a38.png)
